spk/transmission/src/wizard/install_uifile

```diff
@@ -1,69 +1,125 @@
-[{
-    "step_title": "Basic configuration",
-    "items": [{
-        "type": "textfield",
-        "desc": "Download directory",
-        "subitems": [{
-            "key": "wizard_download_dir",
-            "desc": "Download directory",
-            "defaultValue": "/volume1/downloads",
-            "validator": {
-                "allowBlank": false,
-                "regex": {
-                    "expr": "/^\\\/volume\\w*[0-9]{1,2}\\\/[^<>: */?\"]*/",
-                    "errorText": "Path should begin with /volumename?/ where volumename can be 'volume' or also 'volumeUSB' and ? is the volume number (1-99)."
-                }
+[
+    {
+        "step_title": "Basic configuration",
+        "items": [
+            {
+                "type": "combobox",
+                "desc": "Please select a volume to use for the download folder",
+                "subitems": [
+                    {
+                        "key": "wizard_volume",
+                        "desc": "volume name",
+                        "displayField": "display_name",
+                        "valueField": "volume_path",
+                        "editable": false,
+                        "mode": "remote",
+                        "api_store": {
+                            "api": "SYNO.Core.Storage.Volume",
+                            "method": "list",
+                            "version": 1,
+                            "baseParams": {
+                                "limit": -1,
+                                "offset": 0,
+                                "location": "internal"
+                            },
+                            "root": "volumes",
+                            "idProperty": "volume_path",
+                            "fields": [
+                                "display_name",
+                                "volume_path"
+                            ]
+                        },
+                        "validator": {
+                            "fn": "{console.log(arguments);return true;}"
+                        }
+                    }
+                ]
+            },
+            {
+                "type": "textfield",
+                "desc": "Download shared folder (using the volume chosen above)",
+                "subitems": [
+                    {
+                        "key": "wizard_download_dir",
```

Comment:
Aaah, that is the trick, to use separate wizard variables for the volume and the share name!

but this should work in the demoservice too.

Reply:
Only the folder name is used in the worker (not the full path), maybe we need to use `basename` before passing it on to `jq --arg share [..]`

Reply:
Example from documentation:
```json
"data-share": {
    "shares": [{
        "name": "music",
        "permission": {
             "ro": ["AudioStation"]
         }
    }]
}
```

Reply:
@hgy59  whichever way we go, (new wizards or patch to remove `volumeX` before passing the variable on) please remember to remove the if.
https://github.com/SynoCommunity/spksrc/blob/83f850f485c25db3bba65f80a6b95d57cb929b16/spk/demoservice/Makefile#L21-L25